### ISSUE TITLE
Axes toggle on Visualisation doesn't appear to work.

### ISSUE THREAD

**Issue by CompRhys** [materialsproject/crystaltoolkit#198]
I would assume the axes toggle is supposed to turn off the axes indicator in the bottom corner but this doesn't appear to be the case.

![image](https://user-images.githubusercontent.com/26601751/122026397-13889600-cdc2-11eb-9b24-7f22da9cd448.png)
![image](https://user-images.githubusercontent.com/26601751/122026493-2b601a00-cdc2-11eb-906f-4f9b0bda3c2d.png)

**Comment by mkhorton** [comment]
This is true. Some axes functionality broke when we separated the axes out from being part of the scene to being pinned in the corner. Note the deployed version of this app is from last July, there's been a lot of work done in the main branch since then.

However, this code is still not technically released or intended for external use yet, there is no associated publication, tests are still failing, etc. It's being developed in the open but your mileage may vary :-)

By all means send me an email if you want to start using it and have questions.

**Comment by CompRhys** [comment]
Ahh okay, I don't think I am the target user here in terms of what the app offers. I was simply looking for a quick way to get high-ish res structure renders for a presentation. Whenever I use Vesta I always seem to get really poor looking structures with super shiny surfaces and pixelated lines. 

Along the directions above it would be great to be able to vary the camera position numerically but I am not sure how easy it could be implemented

**Comment by mkhorton** [comment]
Ah fair enough. It's decent enough for this, but not as user friendly as it could be right now.

One of the design goals of Crystal Toolkit was to give faithful representations of pymatgen objects, e.g. if you have a specify set of bonds you want to display, or color-code by a specific site property etc. The rendering quality is somewhat improved, but VESTA is more performant.

For visualization purposes, one of the nice aspects of Crystal Toolkit is the Jupyter integration, when you have a visualization you like it can be saved to JSON (.ctk.json), to open up separately later within Jupyter.

**Comment by mkhorton** [comment]
Also just to add -- the app itself, app.crystaltoolkit.org, is intended as a demo of the framework, not the end product in and of itself.
